src/main/java/commands/Greeting.java

```diff
@@ -0,0 +1,13 @@
+package commands;
+
+import constants.constant;
+
+public class Greeting {
+    public static void greeting() {
```

Comment:
When naming a function, verbs should be used. In this case, I feel that this greeting can be made as a constant variable instead of putting it in a function in a separate class on its own.